src/components/themed/SimpleTextInput.tsx

```diff
@@ -52,12 +52,18 @@ export interface SimpleTextInputProps extends MarginRemProps {
   secureTextEntry?: boolean // Defaults to 'false'
   testID?: string
 
-  // Unless 'autoFocus' is passed explicitly in the props, Search Bars 'autoFocus' and 'regular' text inputs don't.
+  /** Unless 'autoFocus' is passed explicitly in the props, Search Bars
+  'autoFocus' and 'regular' text inputs don't. */
   autoFocus?: boolean // Defaults to 'true'
 
-  // Unless 'blurOnClear' is passed explicitly in the props, Search Bars calls 'blur' when cleared and text inputs don't call 'blur' when cleared.
+  /** Unless 'blurOnClear' is passed explicitly in the props, Search Bars calls
+   * 'blur' when cleared and text inputs don't call 'blur' when cleared. */
   blurOnClear?: boolean // Defaults to 'false'
 
+  /** Manually control the whether the input appears selected. Visual only,
+   * mutually exclusive with the text input's true blur/focus state. */
```

Comment:
fixup to the grammar

```js
/** Manually control whether the input appears selected. Visual only; mutually exclusive with the text input's true blur/focus state. */
```

Reply:
ChatGPT said this was fine the way it was

Reply:
ChatGPT is wrong.